src/test/scala/io/iohk/ethereum/blockchain/sync/FastSyncBlockBodiesRequestHandlerSpec.scala

```diff
@@ -83,6 +83,7 @@ class FastSyncBlockBodiesRequestHandlerSpec extends FlatSpec with Matchers {
     val fastSyncBlockBodiesRequestHandler: ActorRef =
       parent.childActorOf(SyncBlockBodiesRequestHandler.props(
         peer,
+        1.second,
```

Comment:
it is not test timeout, it is more of configuration value for SyncBlockBodiesRequestHandler actor, I will put it in named constant